examples/launchToken/index.mjs

```diff
@@ -0,0 +1,12 @@
+import { loadStdlib } from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+
+const stdlib = loadStdlib();
+const [ accA, accB, accC, accD ] = await stdlib.newTestAccounts(4, stdlib.parseCurrency(100));
+
+await stdlib.launchToken(accA, 'Gil', 'GIL', {
+  clawback: accB,
+  freeze: accC,
+  reserve: accD,
+  defaultFrozen: true,
+});
```

Comment:
Can this test then check the on-chain metadata of the token to demonstrate that those addresses are in fact set correctly?